app/services/clients/zendesk.client.js

```diff
@@ -11,33 +10,26 @@ const zendeskClient = zendesk.createClient({
   proxy: process.env.http_proxy
 })
 
-module.exports = {
+function createTicket (opts) {
+  zendeskClient.tickets.create({
```

Comment:
I believe this promise will need to be returned to the caller of the function for the async/ await error handling above to correctly propagate 
```suggestion
  return zendeskClient.tickets.create({
```

Reply:
Adding `return` did not work.
So we reverted the `zendesk.client.js` back to how it was.

Reply:
Actually it does work.  I have updated the code.
Thanks